test/browser/sauce-browsers.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "browserName": "microsoftedge",
+    "platform": "WIN10",
+    "version": "20.10240"
```

Comment:
Maybe, though why would we be any different than Chrome/Firefox?

I've got  a thread going with them to change the version from `20.x` to `12.x` based on our [blog guidance](http://blogs.windows.com/msedgedev/2015/09/21/understanding-versions-in-an-evergreen-browser/).